dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/web/EndpointUtils.java

```diff
@@ -195,4 +195,27 @@ public static String computeTokenHash(String lookupKeyParam, String algorithm) {
         return lookupKey;
     }
 
+    // fix any unescaped ". Doesn't handle bs,ff,\n,\r,\t \Uxxxx, \\, \/
+    public static String escapeQuotesForJson(String in) {
```

Comment:
Methods like this are prime examples of ones that could use unit tests.

Reply:
I'll add one in ghe.